README.md

```diff
@@ -58,6 +58,44 @@ default unless you set the `QT_API` environment variable.
 * `pyside6` (to use PySide6).
 
 
+### Methods, helpers and QtPy namespace specifics
+
+As part of providing compatibility for the different Python Qt bindings available, QtPy does some modules mapping and exposes a `compat` module. Bellow you can check some elements to take into account when using QtPy.
+
+#### Module aliases and constants
+
+* `QtCore.pyqtSignal`, `QtCore.pyqtSlot` and `QtCore.pyqtProperty` (available on PyQt5/6) are replaced by `QtCore.Signal`, `QtCore.Slot` and `QtCore.Property` following the Qt5 module layout.
+
+* The Qt version being used can be checked with `QtCore.__version__` (instead of `QtCore.QT_VERSION_STR`) as well as from `qtpy.QT_VERSION`.
+
+* For PyQt6 enums, unscoped enum access was added by 
+promoting the enums of the `QtCore`, `QtGui`, `QtTest` and `QtWidgets` modules.
+
+* Compatibility is added between the `QtGui` and `QtOpenGL` modules for `QOpenGL*` classes.
+
+* To check the current PySide2/6 binding version you can use `qtpy.PYSIDE_VERSION`. If PySide is not being used as the binding the value returned will be `None`.
```

Comment:
```suggestion
* To check the current PySide2/6 binding version, you can use `qtpy.PYSIDE_VERSION`. If PySide is not being used as the binding, the value will be `None`.
```

Add two missing commas and use correct terminology (a callable returns, while an attribute simply has a value)